scripts/release-profile.ts

```diff
@@ -1,13 +1,15 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node
```

Comment:
@perunt Could you please take a look at this file updates and let me know if it looks ok to you?